sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16_math_functions.asciidoc

```diff
@@ -103,286 +103,559 @@ then it supports the `bfloat16` math functions described in the next section.
 
 === Math Functions
 
-The following functions are only available when `T` is `bfloat16` or
-`sycl::marray<bfloat16, {N}>`, where `{N}` means any positive value of
-`size_t` type.
+The following functions are only available for scalar `bfloat16`, `marray`,
+and `vec` of type `bfloat16`.
```

Comment:
This sentence seems redundant now because you describe the constraints for each function below.  Is there a reason to keep it?

Reply:
Yes. I've removed that in https://github.com/intel/llvm/pull/14002/commits/21b6ba84c64b2a4ce67b6fad30efc8e533794d35